libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -110,18 +110,26 @@ void AC_AttitudeControl::set_throttle_out_unstabilized(float throttle_in, bool r
 // Ensure attitude controller have zero errors to relax rate controller output
 void AC_AttitudeControl::relax_attitude_controllers()
 {
+    // Initialize the attitude variables to the current attitude
     // TODO add _ahrs.get_quaternion()
     _attitude_target_quat.from_rotation_matrix(_ahrs.get_rotation_body_to_ned());
```

Comment:
The code seems to have no problem, but could you explain how this new logic should make a difference?

Reply:
```_attitude_target_euler_rate``` is not initialized in the original code.

Reply:
Okay!